packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -182,16 +233,30 @@ contract LockedGold is
       "Sender must be registered with Account.createAccount to lock or unlock"
     );
     Balances storage account = balances[msg.sender];
+
```

Comment:
Don't you need to call `updateDelegatedAmount` first?

Reply:
Thanks !